include/cute/atom/copy_traits_xe.hpp

```diff
@@ -667,8 +667,8 @@ struct Copy_Traits_<XE_2D_U8x2x64_LD_N, args_t...>
     : XE_2D_LD_Unpack<XE_2D_U8x2x64_LD_N, args_t...> {
   using ThrID = Layout<_16>;
   // Map from (src-thr,src-val) to bit
-  using SrcLayout = Layout<Shape <_16,_8>,
-                           Stride< _0,_1>>;
+  using SrcLayout = Layout<Shape <_16,Shape <_16,  _2,  _2>>,
+                           Stride<_16,Stride< _1,_256,_512>>>;
```

Comment:
Stride between workitems in subgroup should be _0 in SrcLayout for all CopyTraits (workitems in a subgroup all pass coordinates of the first element to the instruction)